Common/UnitDefinitions/HeatFlux.json

```diff
@@ -116,6 +116,30 @@
           "Abbreviations": [ "kcal/h·m²" ]
         }
       ]
+    },
+    {
+      "SingularName": "PoundForcePerFootSecond",
+      "PluralName": "PoundsForcePerFootSecond",
+      "FromUnitToBaseFunc": "x*1.459390293720636e1",
+      "FromBaseToUnitFunc": "x/1.459390293720636e1",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "lbf/(ft·s)", "lb/(ft·s)" ]
```

Comment:
Is `lb` (not `lbf`) a valid abbreviation for this? I've always thought it was to distinguish mass from force, similar to `kgf` vs `kg`.

Reply:
I think you're right. I have seen it listed this way a few places, but I think I'll remove it. Otherwise it's too easy to confuse with lbm/(ft·s) which is for dynamic viscosity.